packages/sdk/src/lib/dataProtectorCore/getGrantedAccess.ts

```diff
@@ -46,6 +46,7 @@ export const getGrantedAccess = async ({
         app: vAuthorizedApp || 'any',
         requester: vAuthorizedUser || 'any',
         isRequesterStrict: vIsUserStrict,
+        isAppStrict: true,
```

Comment:
pass isAppStrict as a parameter to the function (use false as the default value) then validate:

const vIsAppStrict = booleanSchema()
  .label('isAppStrict')
  .validateSync(isAppStrict);

then call fetchDatasetOrderBook, passing isAppStrict = vIsAppStrict.

Reply:
apprestrict address zero should not occur, the current implementation seems OK